references/sdk.md

```diff
@@ -164,6 +164,29 @@ Create new algo asset.
 If `exist_ok` is true, `AlreadyExists` exceptions will be ignored and the
 existing asset will be returned.
 
+## add_composite_algo
+```python
+Client.add_composite_algo(self, data, timeout=False, exist_ok=False)
+```
+Create new composite algo asset.
+
+`data` is a dict object with the following schema:
+
+```
+{
+    "name": str,
+    "description": str,
+    "file": str,
+    "permissions": {
+        "public": bool,
+        "authorizedIDs": list[str],
```

Comment:
I'm sorry, I was confused, these ought to be in snake_case (since they are input values)